modules/apps/frontend-token/frontend-token-definition-test/src/testIntegration/java/com/liferay/frontend/token/definition/test/FrontendTokenDefinitionRegistryTest.java

```diff
@@ -90,37 +90,48 @@ public void tearDown() throws PortalException {
 
 	@Test
 	public void testGetFrontendTokenDefinitions() {
+		List<FrontendTokenDefinition> frontendTokenDefinitionsList =
+			_frontendTokenDefinitionRegistry.getFrontendTokenDefinitions(
+				_layoutSet.getCompanyId());
+
+		Assert.assertTrue(
+			ListUtil.exists(
+				frontendTokenDefinitionsList,
+				item -> Objects.equals(
+					item.getThemeId(), _THEME_ID_CLIENT_EXTENSION_ENTRY)));
+
 		Assert.assertTrue(
 			ListUtil.exists(
-				_frontendTokenDefinitionRegistry.getFrontendTokenDefinitions(
-					_layoutSet.getCompanyId()),
-				frontendTokenDefinition -> Objects.equals(frontendTokenDefinition.getThemeId(), _THEME_ID_CLIENT_EXTENSION_ENTRY)));
+				frontendTokenDefinitionsList,
+				item -> Objects.equals(
+					item.getThemeId(), _THEME_ID_LAYOUT_SET)));
 	}
 
 	@Test
-	public void testGetThemeCSSCETFrontendTokenDefinition() {
+	public void testGetThemeBundleFrontendTokenDefinition() {
+		_layoutSet.setLayoutSetId(RandomTestUtil.randomLong());
+
 		_assertFrontendTokenDefinition(
 			_frontendTokenDefinitionRegistry.getFrontendTokenDefinition(
 				_layoutSet),
-			_THEME_ID_CLIENT_EXTENSION_ENTRY);
+			_THEME_ID_LAYOUT_SET);
 	}
 
 	@Test
-	public void testGetThemeFrontendTokenDefinition() {
-		_layoutSet.setLayoutSetId(-1);
-
+	public void testGetThemeCSSCETFrontendTokenDefinition() {
 		_assertFrontendTokenDefinition(
 			_frontendTokenDefinitionRegistry.getFrontendTokenDefinition(
 				_layoutSet),
-			_THEME_ID_LAYOUT_SET);
+			_THEME_ID_CLIENT_EXTENSION_ENTRY);
```

Comment:
If we're aiming preventing another complain regarding tests naming, we could merge these two tests together and call it `testGetFrontendTokenDefinition`, since both tests use the same method.